packages/cli/src/modules/insights/insights.controller.ts

```diff
@@ -152,7 +152,7 @@ export class InsightsController {
 
 		if (!query.startDate) {
 			return {
-				startDate: DateTime.now().minus({ days: 7 }).toJSDate(),
+				startDate: DateTime.now().minus({ days: 6 }).toJSDate(),
```

Comment:
Might just be worth adding a comment here to explain why we take 6 instead of 7 by default